src/util/request.web.ts

```diff
@@ -10,6 +10,7 @@ export const isBrowser = true;
 
 function omit<T>(obj: T, keys: (keyof T)[]): T {
   const result = {} as T;
+  // @ts-ignore
```

Comment:
why do we use ts-ignore here and also can you add the rule in the comment after ts-ignore

Reply:
We are using it because `Object.keys(obj)` part of the code. `obj` is defined as `obj: T` and Typescript complains about that.